pkg/api/controller.go

```diff
@@ -209,7 +209,7 @@ func (c *Controller) CreatePresignMultipartUpload(w http.ResponseWriter, r *http
 	}
 
 	// generate a new address for the object we like to upload
-	address, err := c.Catalog.GetAddressWithSignature(repository, branch, params.Path)
```

Comment:
It's *NOT* really related to this PR, but -
Anything technical that keeps us from renaming `repository` to `repositoryID`?

Reply:
Not part of this PR